archinstall/lib/configuration.py

```diff
@@ -170,27 +161,23 @@ def preview(selection: str):
 
 		prompt = _(
 			"Do you want to save {} configuration file(s) in the following location?\n\n{}"
-		).format(
-			list(options.keys())[list(options.values()).index(str(save_choice.value))],
-			dest_path.absolute(),
-		)
+		).format(options[str(save_choice.value)], dest_path.absolute())
+
 		save_confirmation = Menu(prompt, Menu.yes_no(), default_option=Menu.yes()).run()
 		if save_confirmation == Menu.no():
 			return
 
 		debug(
-			_("Saving {} configuration files to {}").format(
-				list(options.keys())[list(options.values()).index(str(save_choice.value))],
-				dest_path.absolute(),
-			)
+			_("Saving {} configuration files to {}").format(options[str(save_choice.value)], dest_path.absolute())
```

Comment:
The translation here can actually be removed as otherwise we end up with translated text in the log file which isn't good 

```
debug("Saving {} configuration files to {}".format(options[str(save_choice.value)], dest_path.absolute()))
```

Reply:
Thanks 😄